assignment/scripts/function-practice.js

```diff
@@ -10,74 +10,148 @@ function hello() {
 }
 // Call the function to test
 console.log('Test - should say "Hello World!"', hello());
+console.log(hello());
 
-
-// 2. Function to return an personalized hello, using the `name` argument.
+// 2. Function to return a personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
-function helloName( name ) {
-  return;
+function helloName(name) {
+  console.log(`Hello, ${name}!` );
 }
 // Remember to call the function to test
-
+helloName('Jose');
 
 // 3. Function to add two numbers together & return the result
-function addNumbers( firstNumber ) {
+function addNumbers(firstNumber, secondNumber) {
+  // return firstNumber + secondNumber;
+  return firstNumber + secondNumber;
+}
+let sum = addNumbers(5, 17);
+console.log(sum);
+// or
+console.log(addNumbers(9, 2));
+
+// or 
+
+let _addNumbers = (firstNumber, secondNumber) => {
   // return firstNumber + secondNumber;
+  return firstNumber + secondNumber;
 }
+console.log(_addNumbers(6, 10));
 
+// or
+
+let __addNumbers = (firstNumber, secondNumber) => firstNumber + secondNumber; 
+console.log(__addNumbers(6, 20));
 
 // 4. Function to multiply three numbers & return the result
-function multiplyThree( ){
+function multiplyThree(x, y, z) {
+  return x * y * z;
+}
+let product = multiplyThree(5, 5, 5);
+console.log(product);
+// or 
+console.log(multiplyThree(2, 2, 2));
+
+// or 
 
+let _multiplyThree = (x, y, z) => {
+  return x * y * z;
 }
+console.log(_multiplyThree(3, 3, 3));
 
+// or
+
+let __multiplyThree = (x, y, z) => x * y * z;
+console.log(__multiplyThree(4, 4, 4));
+
+let squared = x => x^2;
```

Comment:
Solid function structure, but `^` is actually kind of a _bitwise operator_, which does something very different from what you want. You wanted `**`.